daemonset/generic-kuberouter-all-features.yaml

```diff
@@ -62,6 +80,9 @@ spec:
       labels:
         k8s-app: kube-router
         tier: node
+      annotations:
+        prometheus.io/port: '20245'
+        prometheus.io/scrape: 'true'
```

Comment:
I think that adding the prometheus annotations along with the metrics-port below is fine, but as for the rest of the changes, I don't think that they represent safe or desired defaults.

For instance, enabling the hostport CNI and binding nodeports on all ips I think are options the users should enable with forethought and consideration.

I would like to merge the changes to documentation that you made below and the prometheus stuff for the default example, but the other items in this file should not be the default IMO.

Reply:
I'd like to leave the comment in the file. Because it's not quite obvious based on the file that the cni config needs some customizations.

Reply:
I think that leaving the comments is fine. Go ahead and revert the other changes that I mentioned and let me know when you're ready for another look.